JanePhotoBrowser/PhotoBrowser/PhotoBrowserFullscreen/PhotoBrowserStyleKit.swift

```diff
@@ -9,6 +9,11 @@
 import UIKit
 
 public class PhotoBrowserIconography: NSObject {
+
+    @available(iOS 13, *)
+    public static func imageOfXIcon() -> UIImage? {
+        return UIImage(systemName: "xmark")
+    }
```

Comment:
Don't override the existing imageOfXIcon(), instead put this inside of an `if #available` inside that function. That way you don't have to put the conditional check in `PhotoBrowserFullscreenViewController.swift` (the one with the comment that it wont work as implemented)